### ISSUE TITLE
rpki-client: Inconsistent RPKI repository caching results

### ISSUE THREAD

**Issue by fjahr** [fjahr/kartograf#19]
We are sometimes getting more RPKI repositories than on other runs. IRR always completes the data but it would be good to understand what is causing this and if there is anything that we can improve on our side.

I opened an issue in the rpki-client github: https://github.com/rpki-client/rpki-client-portable/issues/116

**Comment by job** [comment]
It is best to use the same repository across multiple runs, not start empty every time

**Comment by job** [comment]
Another thing that stands out: the `-P` option is used, this option is only intended for very special use-cases; and the wrong value could result in repositories going missing. Why is `-P` used?

**Comment by fjahr** [comment]
Ok, I think we can try to add something like this for our scheduled runs. We can try to sync once before the we actually run and then just resync with the same cache dir when our run actually starts.

**Comment by job** [comment]
I recommend to somehow sync at least once an hour, this increases the chance to overcome intermittent transient reachability issues repositories might have

**Comment by jurraca** [comment]
iiuc, is this actually an issue? If we want a static set of RPKI repositories, we could use the `-H` flag, which takes a list of domains to use for fetching RPKI data. That way we would have the same repositories every time. But I don't necessarily see the issue with letting `rpki-client` fetch RPKI data for us as it sees fit.

**Comment by fjahr** [comment]
Agree, in the latest runs we have had pretty much no mismatches so it doesn't seem to be an actual problem (anymore). Good to know we have that option with the list but given the recent results I would say we don't do that unless we have to since the list itself would be another thing we have to manage.
